src/runtime/web/io.js

```diff
@@ -16,10 +16,9 @@ function load(type, url) {
       xhr.responseType = 'json';
     }
 
-    xhr.addEventListener('load', function io_onload(e) {
-      if (e.target.status === 200 || e.target.status === 0) {
-        // Sinon.JS's FakeXHR doesn't have the response property
-        resolve(e.target.response || e.target.responseText);
+    xhr.addEventListener('load', e => {
+      if (e.target.status === HTTP_STATUS_CODE_OK) {
```

Comment:
Wasn't `0` used also when accessing `l20n.js` via the `file://` protocol in the browser?